### PR TITLE
Single card titles

### LINKED ISSUE #2649: Error: Group titles must be unique
https://sentry.io/parabol/action-production/issues/884845499/?referrer=github_plugin

```
Error: Group titles must be unique
  File "app:///graphql/mutations/updateReflectionGroupTitle.js", line 58, in resolve
    return standardError(new Error('Group titles must be unique'), {userId: viewerId})

Error: Group titles must be unique
```

### LINKED ISSUE #2850: Error: 3 INVALID_ARGUMENT: The language nl is not supported for entity analysis.
https://sentry.io/organizations/parabol/issues/895876465/?project=107196&referrer=github_plugin

```
Error: 3 INVALID_ARGUMENT: The language nl is not supported for entity analysis.
  File "app:///../../node_modules/grpc/src/common.js", line 91, in Object.exports.createStatusError
    let error = new Error(message);
  File "app:///../../node_modules/grpc/src/client_interceptors.js", line 1204, in Object.onReceiveStatus
    var error = common.createStatusError(status);
  File "app:///../../node_modules/grpc/src/client_interceptors.js", line 568, in InterceptingListener._callNext
    return this.delegate[method_name](args_array[0], next_listener);
  File "app:///../../node_modules/grpc/src/client_interceptors.js", line 618, in InterceptingListener.onReceiveStatus
    this._callNext('onReceiveStatus', [status]);
  File "app:///../../node_modules/grpc/src/client_interceptors.js", line 845, in callback
    first_listener.onReceiveStatus(status);

Error: 3 INVALID_ARGUMENT: The language nl is not supported for entity analysis.
```

AC
- [ ] i added to regex in addEntitiesToReflections

Effort 10.5

### PR DESCRIPTION
**Pull request by mattkrick**
TEST - fix #2649
- [ ] click on a 1-card group in the grouping phase, you can now add a title
- [ ] click again to hide the title, if the title is not user-defined

also fixes #2850

### CODE REVIEW

**Comment by jordanh** [comment]
## Tests
- [ ] click on a 1-card group in the grouping phase, you can now add a title
- [ ] click again to hide the title, if the title is not user-defined

File: src/universal/utils/autogroup/getTitleFromComputedGroup.ts [hunk 1]
```diff
@@ -53,7 +62,13 @@ const getTitleFromComputedGroup = (uniqueLemmaArr, group, reflectionEntities) =>
     // if they get the jist, abort
     if (cumlSalience > SALIENT_THRESHOLD) break
   }
-  if (titleArr.length === 0) return 'New group'
+  if (titleArr.length === 0) {
```
Comment (jordanh) [inline src/universal/utils/autogroup/getTitleFromComputedGroup.ts]: I am going to try my darndest to test this pathway too

File: src/universal/utils/getAuthProviders.ts [hunk 2]
```diff
@@ -9,7 +9,13 @@ const query = graphql`
 `
 
 const getAuthProviders = async (atmosphere: Atmosphere, email: string) => {
-  const res = await fetchQuery<getAuthProvidersQuery>(atmosphere, query, {email})
+  let res
```
Comment (jordanh) [inline src/universal/utils/getAuthProviders.ts]: It's always the external services...

**Comment by jordanh** [comment]
@mattkrick see test notes above...

**Comment by mattkrick** [comment]
column height fixed, merging

**Pull request merged**